src/hotspot/share/services/attachListener.hpp

```diff
@@ -239,11 +245,16 @@ class AttachOperation: public CHeapObj<mtServiceability> {
     virtual int read(void* buffer, int size) = 0;
 
     // Reads unsigned value, returns -1 on error.
-    int read_uint();
+    //
+    // Attach client can make sanity connect/disconnect.
+    // In the case we get "premature EOF" error.
+    // If "may_be_empty" is true, the error is not logged.
```

Comment:
Nits or two minor suggestions:
 - replace: "In the case" => "In that case"
 - replace: "may_be_empty" => may_be_empty

Reply:
Fixed